### PR TITLE
News route

### PR DESCRIPTION
**Pull request by nwiatrek**

### CODE REVIEW

File: news/news.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    regex = re.compile('[A-Za-z]{2,3}-[A-Za-z]{2,3}\b')
+    if regex.match(location) is None:
+        raise Exception('invalid format please look at ISO 3166-1 alpha-2')
+
+
+def sort_by_heat(data):
+    """order by most relevant (heat given from api)"""
+    news = data.get('news')
+    remove_unwanted_keys(news)
+    return sorted(remove_none_heat(news), key=lambda x: x['heat'], reverse=True)
```
Comment (rofranklin) [inline news/news.py:31]: ok, go off

**Review by rofranklin — COMMENTED** [review]
ok the code itself looks good and the tests pass locally but for the love of god pls put in a gitignore bc we have so many .idea and .venv files

File: news/news.py (hunk 1 above)
Comment (rofranklin) [reply news/news.py:31]: also was there a reason we were doing pop and not just parsing out the keys we want? Unless the keys we want are wayyy more than the keys we don't want, it's more efficient.

**Comment by mdyl** [comment]
I'm good with this

File: news/news.py (hunk 1 above)
Comment (nwiatrek) [reply news/news.py:31]: I'm only removing like three keys and i think there are like 15 total

**Comment by nwiatrek** [comment]
I added the git ignore to the branch

**Review by mdyl — APPROVED** [review]

**Pull request closed without merging**